### PR TITLE
docs: improve code snippets for `TabularModalPage` and `InfoModalPage`

### LINKED ISSUE #2950: application-component documentation lists incorrect code example
Have a look here: https://docs.commercetools.com/custom-applications/api-reference/commercetools-frontend-application-components#infomodalpage
The example creates a pageModalState which is never used but refers to a function handleClose that does not exist.

### PR DESCRIPTION
**Pull request by kark**
#### Summary

closes #2950

### CODE REVIEW

**Review by tdeekens — APPROVED** [review]

File: website/src/content/api-reference/commercetools-frontend-application-components.mdx [hunk 1]
```diff
@@ -266,13 +266,13 @@ const Example = () => {
     <InfoModalPage
       title="Lorem ipsum"
       isOpen={true}
```
Comment (CarlosCortizasCT) [inline website/src/content/api-reference/commercetools-frontend-application-components.mdx:268]: I was wondering whether we should also use the modal state for the `isOpen` property.
Perhaps we can initialize the hook with `true` value and use it here so when we call the `closeModal` function, the modal would actually close.

```jsx
const Example = () => {
  const pageModalState = useModalState(true); // modal open by default

  return (
    <InfoModalPage
      title="Lorem ipsum"
      isOpen'{pageModalState.isModalOpen}
      ...
  );
}
```


What do you think?

Comment (kark) [reply website/src/content/api-reference/commercetools-frontend-application-components.mdx]: I added `isModalOpen` but I'm not sure about `useModalState(true)`. I mean only in terms of consistency, as we don't use it in other code snippets 🤔

Comment (CarlosCortizasCT) [reply website/src/content/api-reference/commercetools-frontend-application-components.mdx]: Ok, if we don't use it in other examples, maybe we should aim for consistency, yes.

It just caught my eye as the default value for the state is _false_ so when using `useModalState` with no params the `InfoModalPage` would not be visible until some code calls `pageModalState.openModal()`

**Review by CarlosCortizasCT — APPROVED** [review]
👍

**Review by Rhotimee — APPROVED** [review]

**Pull request merged**